File: app/src/main/java/com/starostinvlad/fan/SplashScreen/SplashScreenContract.java
package com.starostinvlad.fan.SplashScreen;

import com.starostinvlad.fan.BaseMVP.MvpView;

public interface SplashScreenContract extends MvpView {
    void startNextActivity();

    void showUpdateDialog();

    void showProgressDialog();

    void startInstallIntent();
}

File: app/src/main/java/com/starostinvlad/fan/SplashScreen/SplashScreenContract.kt
package com.starostinvlad.fan.SplashScreen

import com.starostinvlad.fan.BaseMVP.MvpView
import kotlin.Throws

interface SplashScreenContract : MvpView {
    fun startNextActivity()
    fun showUpdateDialog()
    fun showProgressDialog()
    fun startInstallIntent()
}